content/tooling/wrangler/webpack.md

```diff
@@ -38,9 +38,9 @@ You can tell Wrangler to use a custom webpack configuration file by setting `web
 
 ```js
 module.exports = {
-  "target": "webworker",
-  "entry": "./index.js",
-  "mode": "production"
+  target: "webworker",
```

Comment:
webworker is the default. Could we remove it?

Reply:
it feels worth calling out, i think; i'd rather folks not try to overwrite it otherwise, but i should also read the rest of this document to see if we call it out elsewhere.

Reply:
hmmm. we do, but also all of our examples include it. we should remove it everywhere if we want to be clearer on this.